lib/IRGen/GenConstant.cpp

```diff
@@ -94,9 +112,11 @@ static llvm::Constant *emitConstantValue(IRGenModule &IGM, SILValue operand) {
     return emitAddrOfConstantString(IGM, SLI);
   } else if (auto *BI = dyn_cast<BuiltinInst>(operand)) {
     switch (IGM.getSILModule().getBuiltinInfo(BI->getName()).ID) {
-      case BuiltinValueKind::PtrToInt: {
-        llvm::Constant *ptr = emitConstantValue(IGM, BI->getArguments()[0]);
-        return llvm::ConstantExpr::getPtrToInt(ptr, IGM.IntPtrTy);
+    case BuiltinValueKind::ZeroInitializer:
+      return emitConstantZero(IGM, BI);
+    case BuiltinValueKind::PtrToInt: {
+      llvm::Constant *ptr = emitConstantValue(IGM, BI->getArguments()[0]);
+      return llvm::ConstantExpr::getPtrToInt(ptr, IGM.IntPtrTy);
```

Comment:
Can you fix the indentation here?

Reply:
Hmm... I just ran the code through `git clang-format` and this was the sub-optimal result. I can either re‑indent all of the case statements to match clang style, or have this change match the rest of the case statements. What would you prefer?

Reply:
I don't mind